hash.go

```diff
@@ -1,16 +1,20 @@
 package bigcache
 
-import "hash/fnv"
+import farm "github.com/dgryski/go-farm"
```

Comment:
If we want use it then it would be nice to have hash bench test proving that go-farm is faster than fnv.